scripts/install_composable.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+echo "Composable Multiplatform installer"
+myos=$(uname -a)
+
+# Ubuntu:
+if echo "$myos" | fgrep -q Ubuntu 2>/dev/null ; then      
+	apt update && apt install -y git clang curl libssl-dev llvm libudev-dev
+# Debian:
+elif echo "$myos" | fgrep -q Debian 2>/dev/null; then
+	echo "Debian Detected"
+	apt update && apt install -y git clang curl libssl-dev llvm libudev-dev
+# Fedora
+elif echo "$myos" | fgrep -q fc32 2>/dev/null; then
+	echo "Fedora detected"
+	dnf update && dnf install clang curl git openssl-devel
+# Arch
+elif echo "$myos" | fgrep -q Arch 2>/dev/null; then
+	pacman -Syu --needed --noconfirm curl git clang
+
+# Nix Os
+elif echo "$myos" | fgrep -q NixOS 2>/dev/null; then
+	nix-env -iA nixpkgs.{git,rustup}
+
+# OpenBSD
+elif echo "$myos" | fgrep -q OpenBSD 2>/dev/null; then
+	pkg_add -uv && pkg_add -iv rust rust-gdb rust-clippy rust-rustfmt 
+
+# FreeBSD
+elif echo "$myos" | fgrep -q FreeBSD 2>/dev/null; then
+	curl https://sh.rustup.rs -sSf | sh
+
+# Darwin
+elif echo "$myos" | fgrep -q Darwin 2>/dev/null; then
+	if ! which brew >/dev/null 2>&1; then
+		/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+	fi
+
+	brew update && brew install openssl cmake llvm
+
+else
+	echo "Invalid operating system detected, do you want to proceed with the installation? yes/no"
+	read response
+	if "no" == $response; then
+		echo "Goodbye"
+		exit 1
+	fi 
+fi
+
+echo "Getting the latest version of Rust"
+# Check if rust is enabled
+if ! which rustup >/dev/null 2>&1; then
+	echo "Installing Rust"
+	curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+	source ~/.cargo/env
+else:
+	rustup update
+fi # new
+
+rustup default stable
+rustup update nightly
+rustup target add wasm32-unknown-unknown --toolchain nightly
+echo "Installing composable node"
+#git clone https://github.com/composablefi/composable
```

Comment:
what this commented out code mean? can we clean it up or write todo if some automation is missing?

Reply:
Was commented out when I was debugging the script, fixed now, good catch